### DISCUSSION TITLE
Is there any way that I could use a field Array inside another field array?

### DISCUSSION THREAD

**Discussion by MarianaAthayde** [react-hook-form/react-hook-form#2338] [Q&A] [answered]
I need to do a form which has an array field with some fields and one of those fields is another array. I would know if there is any way to make it using react-hook-form?
Tys. 😄

**Comment by bluebill1049** [comment, marked as answer]
https://codesandbox.io/s/react-hook-form-usefieldarray-nested-arrays-x7btr

**Comment by MarianaAthayde** [comment]
Thanks a lot. 😄

**Comment by serverizzo** [comment]
Thanks for this answer!
I found that it is a little out of date though. On line 23 of field array 
`ref={register()}` 
needs to be replaced with 
``{...register(`test[${index}].name`)}``
otherwise the form will not update. i.e. you won't be able to print any value in the input. It would just be the default values.
